decimal-macros/examples/test.rs

```diff
@@ -0,0 +1,17 @@
+#![feature(plugin)]
+#![plugin(decimal_macros)]
+extern crate decimal;
+
+fn main() {
+    let a = d128!(0.1);
```

Comment:
This test is different. Let me explain. What you are testing here is that the result of d128!(0.1) is equal to the result of d128!(1) / d128!(10). This might pass even if d128!(10) does not equal a dec128 of decimal value 10. This is why I suggest that the left hand side uses the macro and the right hand side uses conversion from int to dec128 which are trivial.

Also is this test organization going to work with ```cargo test```? If not can you put these tests inside decimal-macros/src/lib.rs inside a test module and annotate appropriately so that it works?